opencti-platform/opencti-front/src/private/components/locations/regions/RegionCreation.tsx

```diff
@@ -113,7 +60,7 @@ interface RegionFormProps {
   inputValue?: string;
 }
 
-const REGION_TYPE = 'Region';
+export const REGION_TYPE = 'Region';
```

Comment:
same `export` question ?

Reply:
That's a leftover from my attempt to generify the CustomFileUploader component.
I have suppressed it in the initial PR.